app/components/on-site-notifications/notification-card.js

```diff
@@ -295,6 +303,93 @@ export default Component.extend(
 			});
 		},
 
+		getMessageWallPostBody(model) {
+			let wallOwner = model.get('metadata') && model.get('metadata.username');
+
+			if (!wallOwner) {
+				wallOwner = this.getMessageWallOwner(model.get('uri'));
+			}
+
+			const isOwnWall = wallOwner === this.usernameMarkup;
```

Comment:
Is username unique? Maybe we should use IDs here.

Reply:
Username is unique across MW.

Reply:
Looks like we can use `const currentUsername = this.wdsOnSiteNotifications.currentUser.name;`